frontend/packages/kubevirt-plugin/src/selectors/vmi-migration/combined.ts

```diff
@@ -0,0 +1,12 @@
+import { getName, getNamespace } from '@console/shared';
+import { getMigrationVMIName, isMigrating } from './selectors';
+
+export const findVMIMigration = (migrations, vmi) => {
```

Comment:
I think we should add type annotations, e.g. `migrations` is treated as an array but the signature doesn't reflect this fact.

I'm OK doing this in a follow-up. When adding new code, always try to make use of proper TypeScript.

Reply:
added the types. Although `K8sResourceKind` instead of a `Migration` because we don't have the  type yet (waits for type generation PR)